ChangeLog

```diff
@@ -55,11 +55,16 @@
     * Fix: fixed crash when passing an empty string to cr_log_info (Ersikan).
     * Fix: fixed compilation errors on macOS (Kare Nuorteva, László Várady).
     * Fix: fixed --debug flag aborting runner with gdbserver >= 8.3.
+    * Fix: fixed gdb stepping issue with '--debug' on macOS (Apple silicon).
     * Misc: various documentation fixes (Oleksii Vilchanskyi, Sam Zaydel, Karim
       Dridi, Florent Poinsard, Tim Gates, László Várady).
     * Misc: various updates to dependencies (László Várady).
     * Misc: various build system fixes (Tomasz Sieprawski, László Várady).
     * Misc: various fixes to the autotools example (Bruno Belany).
+    * Misc: build system fixes and release automation.
+    * Misc: fixed typo in readme (Japroz Saini).
```

Comment:
let's fold that into the `Misc: various documentation fixes` entry line 59

Reply:
Done, thanks for the note.